src/app/features/genes/components/gene-resources/gene-resources.component.ts

```diff
@@ -62,7 +62,7 @@ export class GeneResourcesComponent implements OnInit {
         description:
           'View dementia-related publication information for this gene on PubAD.',
         linkText: 'Visit PubAD',
-        link: `https://adexplorer.medicine.iu.edu/pubad/external/${this.gene?.ensembl_gene_id}`,
+        link: 'https://adexplorer.medicine.iu.edu/pubad',
```

Comment:
What do you think about creating the link with a method, rather than updating the link after creating `this.additionalResources`? e.g. something like this -- 

```TS
function createPubAdLink(hgnc_symbol) {
  let url = 'https://adexplorer.medicine.iu.edu/pubad'
  if (hgnc_symbol) {
    url += `/external/${hgnc_symbol}`
  }
  return url
}

this.additionalResources = [
....
        link: createPubAdLink(this.gene?.hgnc_symbol)
....
```

Reply:
Unfortunately, I don't think you can do that within an array assignment.

Reply:
Good idea.  I can write the function and call it inside of the assignment: ${ createPubAdLink() }